src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -733,6 +733,7 @@ public function testEnvInId()
             PsrContainerInterface::class => true,
             ContainerInterface::class => true,
             'baz_%env(BAR)%' => true,
+            'bar_%env(BAR)%' => true,
```

Comment:
the added entries are the same as on 4.0/master: we now list the public-by-private legacy services as removed, even if they're not really for BC in 3.4.

Reply:
won't they appear as removed even if they are not in 4.0?

Reply:
wdyt? they *are* in 4.0

Reply:
definition isn't removed if injected twice or more, is it? If it isn't, wouldn't `getRemovedIds()` return an id that has not been removed?